tests/lambda_endpoints/test_ialirt_archive.py

```diff
@@ -0,0 +1,46 @@
+"""Test the I-Alirt archive lambda function."""
+
+from datetime import datetime, timedelta
+
+import pytest
+
+from sds_data_manager.lambda_code.IAlirtCode.ialirt_archive import lambda_handler
+
+
+@pytest.fixture
+def populate_algorithm_table(setup_dynamodb):
+    """Populate the algorithm table with test entries."""
+    algorithm_table = setup_dynamodb["algorithm_table"]
+    now = datetime.utcnow()
```

Comment:
It is deprecated, so we should be moving away from this call.
https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow